docs/workers.md

```diff
@@ -238,6 +238,12 @@ expressions:
     ^/_matrix/client/(api/v1|r0|v3|unstable)/joined_rooms$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/search$
 
+    # Encryption requests
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/keys/query$
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/keys/changes$
```

Comment:
These two endpoints aren't actually available at `api/v1`, seems silly to route those anywhere...

Reply:
It seems like there's quite a few endpoints that this is true for...

Reply:
arguably we shouldn't be documenting any `api/v1` or `unstable` prefixes. Any clients using them are buggy...